tests/ui/asm/invalid_clobber_abi.rs

```diff
@@ -0,0 +1,18 @@
+// test that asm!'s clobber_abi doesn't accept non-string literals
+// https://github.com/rust-lang/rust/issues/112635
+
+use std::arch::asm;
+
+fn main() {
+    unsafe {
+        asm!("", clobber_abi(1)); //~ ERROR expected string literal
```

Comment:
can you do `clobber_abi()` without any args too?

Reply:
There's a test in the x86_64 specific tests that handles this:
https://github.com/rust-lang/rust/blob/331249a94930adb3d2a7243d946733b63324b2d6/tests/ui/asm/x86_64/parse-error.rs#L41-L42

Reply:
hm now that I think about it, that file should be brought out to the main asm tests and this file's contents added to it